ddl/ddl_api.go

```diff
@@ -384,6 +384,10 @@ func buildColumnAndConstraint(
 	tblCharset string,
 	tblCollate string,
 ) (*table.Column, []*ast.Constraint, error) {
+	if colName := colDef.Name.Name.L; colName == "_tidb_rowid" {
```

Comment:
Maybe we should use `model.ExtraHandleName.L`.

Reply:
Done.